src/time_travel/patchers/datetime_patcher.py

```diff
@@ -202,28 +195,43 @@ def start(self):
         local_names = tuple(real_name for real_name, _, _, _ in to_patch)
         real_id_to_fake = {id(real): fake for _, real, _, fake in to_patch}
 
-        modules = [sys.modules[name] for name in self.patched_modules] if \
-            self.patched_modules else [
+        # Create the list of all modules to search for datetime and date
+        # classes.
+        if self.patched_modules:
+            # If only a given list of modules is required to be patched
+            modules = [sys.modules[name] for name in self.patched_modules] 
+        else:
+            # Patch on all loaded modules
+            modules = [
                 module for mod_name, module in sys.modules.items() if
-                mod_name is not None and module is not None and
+                mod_name is not None and
+                module is not None and
                 hasattr(module, '__name__') and
+                # Don't patch inside this module,
+                # or inside the original module.
                 module.__name__ not in ('datetime', 'datetime_patcher') 
```

Comment:
Is it possible to do `module.__name__ not in ('datetime', __name__)`?

Reply:
cools.